kubeproxy/config/crd/bases/addons.x-k8s.io_kubeproxies.yaml

```diff
@@ -12,20 +14,20 @@ spec:
     listKind: KubeProxyList
     plural: kubeproxies
     singular: kubeproxy
-  scope: ""
+  scope: Namespaced
```

Comment:
This is a notable change -- I think it does make sense.
The Namespace of the CustomResource can be used to patch the manifests.
Most clusters will only have 1 instance of this CR, but containing it within a namespace like `kube-system` could make RBAC and ownership more clear.
You *could* manage kube-proxy daemonsets for different Node Groups in separate Namespaces.

Reply:
This is inline with the existing operators (eg coredns). I think I generated this CRD incorrectly by not picking Namespaced from the onset. 

Really great point about different node groups.